Source/Core/DolphinWX/VideoConfigDiag.cpp

```diff
@@ -284,6 +284,9 @@ static wxString true_color_desc =
     wxTRANSLATE("Forces the game to render the RGB color channels in 24-bit, thereby increasing "
                 "quality by reducing color banding.\nIt has no impact on performance and causes "
                 "few graphical issues.\n\n\nIf unsure, leave this checked.");
+static wxString vertex_rounding_desc =
+    wxTRANSLATE("Round 2D vertices to whole pixels.  Fixes some "
+                "games at higher internal resolutions.\n\nIf unsure, leave this unchecked.");
```

Comment:
I tested this and yeah, the box doesn't get modified when the checkbox is disabled.  I updated the text, please take a look and let me know how you think it reads.